client_taxi_driver/shared/src/commonMain/kotlin/presentation/map/MapScreenUiState.kt

```diff
@@ -0,0 +1,19 @@
+package presentation.map
+
+import presentation.base.ErrorState
+
+data class MapScreenUiState(
+    val isLoading: Boolean = true,
+    val error: ErrorState? = null,
+    val userName: String = "Kamel Mohamed!",
```

Comment:
the default value of the userName should be an empty string, and this value "Kamel Mohamed!" should return from the fake gateway

Reply:
Done